sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/DependencyManager.groovy

```diff
@@ -62,13 +62,13 @@ class DependencyManager {
 
     private void setupCreateNotification(){
         FetchSubscriberDbConnector connector = new FetchSubscriberDbConnector(DatabaseSession.getInstance())
-        createNotificationPresenter = new CreateNotificationPresenter(notificationPerSubscriber)
+        createNotificationPresenter = new CreateNotificationPresenter(notifications)
         createNotification = new CreateNotification(connector, createNotificationPresenter)
         createNotificationController = new CreateNotificationController(createNotification)
     }
 
     private void setupSendEmail(){
-        emailGenerator = new EmailGenerator(notificationPerSubscriber)
+        emailGenerator = new EmailGenerator(notifications)
```

Comment:
why do you provide the notification list via the constructor and not trigger it via method call during runtime (eg in the presenter)